modules/homestakeros/options.nix

```diff
@@ -320,6 +320,28 @@
       };
 
       addons = {
+        ssv-node = {
+          privateKeyFile = mkOption {
+            type = types.nullOr types.path;
+            default = "/var/mnt/secrets/ssv/ssv_operator_key";
+            description = "Path to the private SSV operator key.";
+          };
+          execEndpoint = mkOption {
+            type = types.str;
+            default = "ws://127.0.0.1:8551";
+            description = "WebSocket URL of the execution node to connect to.";
+          };
+          consEndpoint = mkOption {
+            type = types.str;
+            default = "http://127.0.0.1:5052";
+            description = "HTTP URL of the consensus node to connect to.";
+          };
+          dataDir = mkOption {
+            type = types.path;
+            default = "/var/mnt/ssv";
```

Comment:
maybe mnt/addons/ssv? similar change to the private key field

Reply:
Done, I need to remember to add mention of this at the [HomestakerOS/getting_started.md](https://github.com/ponkila/HomestakerOS/blob/311f36a719f552719649a2e3ed35743100b7c6ae/docs/getting_started.md) since the `addons` directory would represent an additional btrfs volume.